roles/smallfile-bench/README.md

```diff
@@ -0,0 +1,38 @@
+Role Name
```

Comment:
Boiler plate file, can be removed or update to reflect the smallfile workload.

Reply:
Remove the file altogether.